rust/cubestore/cubestore/src/metastore/table.rs

```diff
@@ -32,6 +33,10 @@ pub struct Table {
     #[serde(default)]
     unique_key_column_indices: Option<Vec<u64>>,
     #[serde(default)]
+    aggregate_column_indices: Option<Vec<u64>>,
```

Comment:
This is a great time to start documenting our core datastructures a bit. Something like "Vector of aggregate index dimension indices". Or are these aggregation exprs, e.g. `fn(field)` ?

2 more questions:

* Why `Option<Vec<...>>` instead of simply `Vec<...>` with [] as default?
* What is the meaning of a tuple of 2 strings instead of a single string (column name?) or number (column index?) If there is a specific reason for two fields, please create a struct with named fields instead of a tuple.

While at it, is there a chance you could document all the other fields in this struct and then we can have @paveltiunov perform a sanity check?

Reply:
This is one of those places that may change in the future. I've been thinking about this for a while, but I don't know yet how it will be more convenient to use these fields in real code